### DISCUSSION TITLE
TheHive

### DISCUSSION THREAD

**Discussion by ernstdieter** [Security-Onion-Solutions/securityonion#8347] [Unsupported Versions] [answered]
Hi guys,
I still run SO 2.3.91 because of TheHive.
When I upgrade to newest version I will lose the containers, correct?

Is it possible to use containers direct from TheHive? Has anyone experience with it? 

Best Regards

**Comment by dougburks** [comment, marked as answer]
We no longer include or support TheHive. If you would like to continue using TheHive, you will need to reach out to TheHive community for support. Most of our users have switched to our Cases interface since it is tightly integrated with the rest of Security Onion:
https://docs.securityonion.net/en/2.3/cases.html
